src/physics/base/PhysicsInterface.hh

```diff
@@ -9,35 +9,157 @@
 
 #include "base/Span.hh"
 #include "Types.hh"
+#include "physics/grid/XsGridPointers.hh"
+#include "physics/em/detail/LivermorePE.hh"
+#include "physics/em/detail/EPlusGG.hh"
+#include "physics/material/Types.hh"
 
 namespace celeritas
 {
+//---------------------------------------------------------------------------//
+//! Hardcoded types of grid data
+enum class PhysicsTableType
+{
+    macro_xs,    //!< Interaction cross sections
+    energy_loss, //!< Energy loss per unit length
+    range,       //!< Particle range
+    size_        //!< Sentinel value
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * Energy-dependent model IDs for a single process and particle type.
+ */
+struct ModelGroup
+{
+    Span<const real_type> energy; //!< Energy grid bounds [MeV]
+    Span<const ModelId>   model;  //!< Corresponding models
```

Comment:
Not quite sure I'm understanding correctly, is `energy` a span of size 2 that gives the lower and upper energy grid bounds for all models in the group? Or of size `model.size() + 1`, and gives the bounds for each model?

Reply:
The idea is the latter (but of course it's not implemented yet). I'll add a comment.